cmd/context/use.go

```diff
@@ -93,6 +93,11 @@ Or a Kubernetes context:
 
 func (c *ContextCommand) Run(ctx context.Context, ctxOptions *ContextOptions) error {
 	ctxStore := okteto.ContextStore()
+	if len(ctxStore.Contexts) == 0 {
+		// if the context store has no context stored, set flag to save
```

Comment:
Can we specify that this is just when the command is not a context one, for example (build deploy up)

Reply:
sure!